install-k8s.yml

```diff
@@ -6,8 +6,17 @@
     - runtime
     - download-k8s
     - install-k8s
+    - role: install-cni-plugins
```

Comment:
wondering if we can move this to flannel role

Reply:
I had kept this generic because it needs to run on both master and workers for flannel.